.github/workflows/publish-docker-e2e-service.yaml

```diff
@@ -1,36 +1,19 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+
 
 name: Publish E2E service images
 
 on:
   push:
     branches:
-      - master
-    paths:
-      - 'test/e2e-v2/java-test-service/**'
-      - 'test/Makefile'
+      - skywalking-azure
 
 env:
   SKIP_TEST: true
-  HUB: ghcr.io/apache/skywalking
+  HUB: ghcr.io/keshav8s/skywalking
```

Comment:
this needs update- use cloudscaleinc/skywalking

Reply:
yes, i have updated the yaml file